backend/pong/matches/views.py

```
@@ -42,10 +42,14 @@
                                 {
                                     constants.MatchFields.ID: 1,
                                     constants.MatchFields.ROUND_ID: 1,
+                                    constants.MatchFields.STATUS: matches_constants.MatchFields.StatusEnum.COMPLETED.value,
```

Ruff (0.8.2)

Undefined name `matches_constants`

(F821)